app/fullService/api/getWalletStatus.ts

```diff
@@ -1,23 +1,49 @@
-import type { WalletStatus } from '../../types/WalletStatus.d';
+import { MOBTOKENID } from '../../constants/app';
+import type { Accounts } from '../../types/Account.d';
+import type { WalletStatusV2, WalletStatus } from '../../types/WalletStatus.d';
 import axiosFullService, { AxiosFullServiceResponse } from '../axiosFullService';
+import getAllAccounts from './getAllAccounts';
 
 const GET_WALLET_STATUS_METHOD = 'get_wallet_status';
 
 type GetWalletStatusResult = {
   walletStatus: WalletStatus;
 };
 
+function convertWalletStatusFromV2(
+  status: WalletStatusV2,
+  accounts: Accounts
+): GetWalletStatusResult {
+  return {
+    walletStatus: {
+      accountIds: accounts.accountIds,
+      accountMap: accounts.accountMap,
+      isSyncedAll: status.isSyncedAll,
+      localBlockHeight: status.localBlockHeight,
+      minSyncedBlockIndex: status.minSyncedBlockIndex,
+      networkBlockHeight: status.networkBlockHeight,
+      object: 'wallet_status',
+      totalOrphanedPmob: status.balancePerToken[MOBTOKENID]?.orphaned,
+      totalPendingPmob: status.balancePerToken[MOBTOKENID]?.pending,
+      totalSecretedPmob: status.balancePerToken[MOBTOKENID]?.secreted,
+      totalSpentPmob: status.balancePerToken[MOBTOKENID]?.spent,
+      totalUnspentPmob: status.balancePerToken[MOBTOKENID]?.unspent,
```

Comment:
on `status.balancePerToken[MOBTOKENID]` being undefined, these will be assigned undefined rather than a number. Do they need a default value like in `convertBalanceFromV2` in app/fullService/api/getBalanceForAccount.ts ?